zulipterminal/ui_tools/boxes.py

```diff
@@ -1112,26 +1112,39 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         if not reactions:
             return ""
         try:
+            my_user_id = self.model.user_id
             reaction_stats = defaultdict(set)
             for reaction in reactions:
                 user_id = int(reaction["user"].get("id", -1))
                 if user_id == -1:
                     user_id = int(reaction["user"]["user_id"])
-                reaction_stats[reaction["emoji_name"]].add(user_id)
+                user_name = reaction["user"]["full_name"]
+                if user_id == my_user_id:
+                    user_name = "You"
+                reaction_stats[reaction["emoji_name"]].add((user_id, user_name))
 
             sorted_stats = sorted(
-                (reaction, count) for reaction, count in reaction_stats.items()
+                (reaction, list(count)) for reaction, count in reaction_stats.items()
```

Comment:
I'm not sure why the reactions were sorted here in the first place. From [this](https://chat.zulip.org/#narrow/stream/1-announce/topic/server.20releases/near/1355228) message, the ordering of reactions seem to differ from the webapp.

Also what's the reason for converting the set into a list?

Reply:
To rearrange ``You`` reaction ``user_name`` to the end of users reacted.

Reply:
I think now with the updated PR, the tests are also not a problem. So maybe it's better to display the reactions in the order in which they were reacted instead of alphabetically?